regexp.h

```diff
@@ -0,0 +1,26 @@
+/****************************************************************************
+ * bfs                                                                      *
+ * Copyright (C) 2016-2022 Tavian Barnes <tavianator@tavianator.com>        *
```

Comment:
Dates and author are wrong :)

Reply:
Hmm, what's wrong? I copy & pasted that from some your file. :)

Reply:
The date should be just 2022 for this file, and you wrote it, not me

Reply:
Let's write "Tavian Barnes and contributors", ok? :)

Reply:
Sure